docs/src/openapi-v1.yaml

```
@@ -273,6 +273,56 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /blocks/{blockId}/extrinsics/{extrinsicsIndex}:
+    get:
+      tags:
+      - blocks
+      summary: Get an extrinsic by its extrinsicIndex and block height or hash.
+      description: Returns a single extrinsic.
+      operationId: getExtrinsicsByExtrsinsicIndex
```

```suggestion
      operationId: getExtrinsicByTimepoint
```
If we decide to use the `Timepoint` language